core_backend/app/contents/routers.py

```diff
@@ -30,7 +30,13 @@
     CustomErrorList,
 )
 
-router = APIRouter(prefix="/content", tags=["Content Management"])
+TAG_METADATA = {
+    "name": "Question-answering content management",
+    "description": "Authentication: User login. Manage content and tags to be used for "
+    "question answering",
+}
```

Comment:
Thoughts on dropping the "Question-answering" part of this and the tags one? Makes the line super long which makes the description squish up against it badly when you resize the window to be small. Or we can rethink the inclusion of the descriptions now that we have the little blurb at the top of the page?

Reply:
Cool with it! I'll update